.php_cs.php

```diff
@@ -1,186 +0,0 @@
-    'no_unneeded_control_parentheses' => true,
-    'no_unneeded_curly_braces' => true,
-    'no_unneeded_final_method' => true,
-    'no_unreachable_default_argument_value' => true,
-    'no_unset_cast' => true,
-    'no_unset_on_property' => true,
-    'no_unused_imports' => true,
-    'no_useless_else' => true,
-    'no_useless_return' => true,
-    'no_whitespace_before_comma_in_array' => true,
-    'no_whitespace_in_blank_line' => true,
-    'non_printable_character' => [
-        'use_escape_sequences_in_strings' => false,
-    ],
-    'normalize_index_brace' => true,
-    'object_operator_without_whitespace' => true,
-    'ordered_imports' => true,
-    'phpdoc_add_missing_param_annotation' => true,
-    'phpdoc_no_package' => true,
-    'phpdoc_indent' => true,
-    'phpdoc_inline_tag' => true,
-    'phpdoc_no_access' => true,
-    'phpdoc_no_alias_tag' => true,
-    'phpdoc_no_useless_inheritdoc' => true,
-    'phpdoc_return_self_reference' => true,
-    'phpdoc_scalar' => true,
-    'phpdoc_single_line_var_spacing' => true,
-    'phpdoc_summary' => true,
-    'phpdoc_to_comment' => true,
-    'phpdoc_trim' => true,
-    'phpdoc_trim_consecutive_blank_line_separation' => true,
-    'phpdoc_types' => true,
-    'phpdoc_types_order' => [
-        'null_adjustment' => 'always_last',
-        'sort_algorithm' => 'none',
-    ],
-    'phpdoc_var_annotation_correct_order' => true,
-    'phpdoc_var_without_name' => true,
-    'php_unit_construct' => true,
-    'php_unit_dedicate_assert' => true,
-    'php_unit_expectation' => true,
-    'php_unit_fqcn_annotation' => true,
-    'php_unit_method_casing' => true,
-    'php_unit_mock' => true,
-    'php_unit_no_expectation_annotation' => true,
-    'php_unit_ordered_covers' => true,
-    'php_unit_set_up_tear_down_visibility' => true,
-    'php_unit_test_case_static_method_calls' => ['call_type' => 'self'],
-    'protected_to_private' => true,
-    'psr4' => true,
-    'return_type_declaration' => true,
-    'self_accessor' => true,
-    'semicolon_after_instruction' => true,
-    'set_type_to_cast' => true,
-    'short_scalar_cast' => true,
-    'single_blank_line_before_namespace' => true,
-    'single_class_element_per_statement' => true,
-    'single_line_comment_style' => true,
-    'single_quote' => true,
-    'space_after_semicolon' => [
-        'remove_in_empty_for_expressions' => true,
-    ],
-    'standardize_not_equals' => true,
-    'strict_comparison' => true,
-    'strict_param' => true,
-    'string_line_ending' => true,
-    'standardize_increment' => true,
-    'ternary_operator_spaces' => true,
-    'trailing_comma_in_multiline_array' => true,
-    'trim_array_spaces' => true,
-    'unary_operator_spaces' => true,
-    'void_return' => true,
-    'whitespace_after_comma_in_array' => true,
-];
-
-$config = \PhpCsFixer\Config::create()
-    ->setRiskyAllowed(true)
-    ->setRules($rules);
-
-$finder = \PhpCsFixer\Finder::create()
-    ->in('Classes')
```

Comment:
I think we need to keep the directory configuration so we don't check the `vendor` directory.